src/main/java/org/ays/common/util/validation/CityName.java

```diff
@@ -0,0 +1,40 @@
+package org.ays.common.util.validation;
+
+import jakarta.validation.Constraint;
+import jakarta.validation.Payload;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Annotation to validate cities using {@link CityNameValidator}.
+ */
+@Target(value = ElementType.FIELD)
+@Retention(RetentionPolicy.RUNTIME)
+@Constraint(validatedBy = CityNameValidator.class)
+public @interface CityName {
```

Comment:
Aşağıdaki şekilde daha iyi olmaz mı?
```suggestion
public @interface City {
```

Reply:
Aslında ben de başta o şekilde düşündüm ama RolName gibi bir standartta olmalı düşüncesiyle CityName yaptım. City olarak değiştirebilirim ancak zaten City içinde Filtreleme validasyonu yapacağım değiştirmişken CityForm yada CityNameForm şeklinde değiştirebilirim. Nasıl yapayım? yoksa şimdi City yapayım Filtreleme issuesunu yaparken mi değiştireyim?

Reply:
@filizhelvaci @MenekseYuncu İkisi için ayrı ayrı olmasına gerek var mı? Farkı tam olarak neydi. Yani aslında örnek veriyorum `.` karakterini kabul etmiyorsak ikisinde de etmememiz gerekmez mi?

Reply:
@agitrubard form alanlarında harften sonra kabul ettiğimiz karakterleri(-.,') gibi filtrelemede arama yaparken kelimenin başında yada sonunda olacak şekilde kullanabiliyoruz. O sebeple aynı regexle çözemiyoruz.

Reply:
Bu durumda `@CityForm` yapalım, diğerini de `@CityFilter` yapabiliriz bu sayede